src/components/DataTable/index.jsx

```diff
@@ -67,40 +72,48 @@ class DataTable extends Component {
       sortDirection,
       headers,
       items,
+      title,
     } = this.props;
     const colSpan = (headers && headers.length) || 0;
 
     return (
-      <Table className={classes.table} aria-labelledby="tableTitle">
-        {headers && (
-          <TableHead>
-            <TableRow>
-              {headers.map(header => (
-                <TableCell key={`table-header-${header}`}>
-                  <TableSortLabel
-                    id={header}
-                    active={header === sortByHeader}
-                    direction={sortDirection || 'desc'}
-                    onClick={this.handleHeaderClick}>
-                    {header}
-                  </TableSortLabel>
-                </TableCell>
-              ))}
-            </TableRow>
-          </TableHead>
+      <Fragment>
+        {title && (
+          <Typography variant="subheading" align="center" color="textSecondary">
```

Comment:
* Change the variant to `title` and remove the `align` and `color` props.
* Use the Toolbar component.

```jsx
<Fragment>
  {title && (
    <Toolbar>
      <Typography variant="title">
        {title}
      </Typography>
    </Toolbar>
  )}
  // ...
</Fragment>
```